images/kindnetd/cmd/kindnetd/cni.go

```diff
@@ -76,6 +94,9 @@ const cniConfigTemplate = `
 			]
 		]
 		}
+		{{if .Mtu}},
```

Comment:
if the eth0 interface is not found mtu will be 0, 
and {{if .Mtu}}, will skip this based on the go template docs
```
{{if pipeline}} T1 {{end}}
	If the value of the pipeline is empty, no output is generated;
	otherwise, T1 is executed. The empty values are false, 0, any
	nil pointer or interface value, and any array, slice, map, or
	string of length zero.
	Dot is unaffected.
```

Reply:
Yes, and then it will default to the ptp behavior of being set by the kernel. I thought that's what we wanted https://github.com/kubernetes-sigs/kind/pull/1690#discussion_r445998059 

In the docs it says it is an optional field and will be set by Kernel if its not there: https://github.com/kubernetes-sigs/kind/pull/1690#discussion_r445998059

```
Network configuration reference
name (string, required): the name of the network
type (string, required): "ptp"
ipMasq (boolean, optional): set up IP Masquerade on the host for traffic originating from ip of this network and destined outside of this network. Defaults to false.
mtu (integer, optional): explicitly set MTU to the specified value. Defaults to value chosen by the kernel.
ipam (dictionary, required): IPAM configuration to be used for this network.
dns (dictionary, optional): DNS information to return as described in the Result.
```

Reply:
right, perfect